packages/core-bridge/src/envconfig.rs

```diff
@@ -0,0 +1,196 @@
+use std::collections::HashMap;
+
+use bridge_macros::{TryIntoJs, js_function};
+use neon::prelude::*;
+use serde::Serialize;
+use temporal_sdk_core_api::envconfig::{
+    self, ClientConfig as CoreClientConfig, ClientConfigCodec as CoreClientConfigCodec,
+    ClientConfigProfile as CoreClientConfigProfile, ClientConfigTLS as CoreClientConfigTLS,
+    DataSource as CoreDataSource,
+};
+
+use crate::helpers::{BridgeError, BridgeResult};
+
+pub fn init(cx: &mut ModuleContext) -> NeonResult<()> {
+    cx.export_function("loadClientConfig", load_client_config)?;
+    cx.export_function("loadClientConnectConfig", load_client_connect_config)?;
+    Ok(())
+}
+
+impl From<envconfig::ConfigError> for BridgeError {
+    fn from(e: envconfig::ConfigError) -> Self {
+        Self::TypeError {
+            field: None,
+            message: e.to_string(),
+        }
+    }
+}
+
+#[derive(TryIntoJs, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct ClientConfig {
+    profiles: HashMap<String, ClientConfigProfile>,
+}
+
+impl From<CoreClientConfig> for ClientConfig {
+    fn from(c: CoreClientConfig) -> Self {
+        Self {
+            profiles: c.profiles.into_iter().map(|(k, v)| (k, v.into())).collect(),
+        }
+    }
+}
+
+#[derive(TryIntoJs, Serialize)]
```

Comment:
Why `Serialize`/`serde`? Seems like this is all being transfered using `TryIntoJs`.